vaadin-date-picker-flow-parent/vaadin-date-picker-flow/src/main/java/com/vaadin/flow/component/datepicker/DatePicker.java

```diff
@@ -809,27 +794,6 @@ public void removeThemeVariants(DatePickerVariant... variants) {
                         .collect(Collectors.toList()));
     }
 
-    /**
-     * Returns true if the given feature flag is enabled, false otherwise.
-     * <p>
-     * Exposed with protected visibility to support mocking
-     * <p>
-     * The method requires the {@code VaadinService} instance to obtain the
-     * available feature flags, otherwise, the feature is considered disabled.
-     *
-     * @param feature
-     *            the feature flag.
-     * @return whether the feature flag is enabled.
-     */
-    protected boolean isFeatureFlagEnabled(Feature feature) {
```

Comment:
Due to removing the feature flag checks, there are several unused imports in all component classes:
https://github.com/vaadin/flow-components/blob/74fcaf8a38c18ff009c48e616f19783c07b02cd7/vaadin-date-picker-flow-parent/vaadin-date-picker-flow/src/main/java/com/vaadin/flow/component/datepicker/DatePicker.java#L28-L30

Reply:
Good point, fixed.